### PR TITLE
CNFT1-2244 Login page: About/Our Vision/Get Involved

### PR DESCRIPTION
**Pull request by jlee-enquizit**
## Description

Here we insert the About/Our Vision/Get Involved pages into the WelcomeLayout.
Also small router fix
image

## Tickets

* [CNFT1-2244](https://cdc-nbs.atlassian.net/browse/CNFT1-2244)
* [CNFT1-2262](https://cdc-nbs.atlassian.net/browse/CNFT1-2262)
* [CNFT1-2263](https://cdc-nbs.atlassian.net/browse/CNFT1-2263)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT1-2244]: https://cdc-nbs.atlassian.net/browse/CNFT1-2244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-2262]: https://cdc-nbs.atlassian.net/browse/CNFT1-2262?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-2263]: https://cdc-nbs.atlassian.net/browse/CNFT1-2263?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hclarkEnq** [comment]
question: Should this component design be broken down into components for each subpage? foreseeing potential future changes that might be easier to handle if thats the case, perhaps separating the html structure from the content itself as well?

**Comment by adamloup-enquizit** [comment]
> question: Should this component design be broken down into components for each subpage? foreseeing potential future changes that might be easier to handle if thats the case, perhaps separating the html structure from the content itself as well?

Good suggestion, smaller single purpose components are easier to maintain.  Also, the number of tabs and their contents are likely to change in the near future.

**Review by hclarkEnq — DISMISSED** [review]

**Review by adamloup-enquizit — CHANGES_REQUESTED** [review]
I'm not sure if those tests are valuable outside of coverage, I get it 😜   

Just noticing that `welcomeLayout` is in the wrong spot, we'll have to move that outside of this PR.

File: apps/modernization-ui/src/welcomeLayout/welcomeRouting.tsx [hunk 1]
```diff
@@ -8,18 +9,15 @@ const routing = {
         { index: true, element: <Navigate to="about" /> },
         {
             path: 'about',
-            // to be added with actual elements/component
-            element: <h1>About</h1>
+            element: <Landing path="about" />
```
[the remark this answers is not in the document]
Comment (jlee-enquizit) [reply apps/modernization-ui/src/welcomeLayout/welcomeRouting.tsx:12]: To maintain consistency for all the different components it ingests.
No good?

**Review by adamloup-enquizit — APPROVED** [review]
🌚<sup> <sup>🚀</sup></sup>

**Review by hclarkEnq — APPROVED** [review]

**Pull request merged**